CHANGES.txt

```diff
@@ -1,3 +1,32 @@
+0.3
+---
+
+- Python 3 is now supported! This is possible thanks to python-ldap supporting
+  python 3
+
+- BREAKING CHANGE: pyramid_ldap now depends on python-ldap >= 3.0, which
+  supports python 2 and 3. If you upgrade pyramid_ldap without upgrading
+  python-ldap, you will see failures where python-ldap receives text arguments
+  instead of bytes. See below for more information.
+
+- python-ldap >= 3.0 stopped accepting ``bytes`` type arguments for many
+  ``LDAPObject`` methods. We now use the ``pyramid.compat.text_`` function to
+  support text arguments for python 2 and 3.
+
+- The  method typically deals with
```

Comment:
What method? I am not sure I understand this changelog entry.

Reply:
One example is https://python-ldap.readthedocs.io/en/latest/reference/ldap.html#ldap.LDAPObject.search_s, but it was a general update that affected a number of methods on their LDAPObject class. The interface before was clunky (having to convert arguments to bytes)-- this was actually a major improvement.

However, it also means pyramid_ldap is broken on master. If you install it as-is you'll get the newer pyldap (now a virtual package for python-ldap) and python-ldap with the changed API, and it'll raise errors where it expects type `str` now.

Reply:
@ericrasmussen I was merely commenting on the fact that it's a stand-alone bullet point entry in the CHANGES.txt with 0 context of what method or what is going on.